electrum/gui/qt/installwizard.py

```diff
@@ -197,10 +187,41 @@ def __init__(self, config: 'SimpleConfig', app: QApplication, plugins: 'Plugins'
         outer_vbox.addLayout(hbox)
         outer_vbox.addLayout(Buttons(self.back_button, self.next_button))
         self.set_icon('electrum.png')
+        self._set_gui_text()
         self.show()
         self.raise_()
         self.refresh_gui()  # Need for QT on MacOSX.  Lame.
 
+    def _set_gui_text(self):
+        self.setWindowTitle('ELCASH Wallet  -  ' + _('Install Wizard'))
+        self.back_button.setText(_('Back') if self.can_go_back() else _('Cancel'))
+        self.next_button.setText(_("Next"))
+        self.please_wait.setText(_("Please wait..."))
+
+    def select_and_save_language(self):
+        """Method for selecting and saving language in config file as {'language': <language-abbreviation: str>}"""
+        filtered_languages = dict(filter(lambda item: item[1] != 'Default', languages.items()))
```

Comment:
wouldn't it be more readable?
`{key: value for key, value in languages.items() if value != 'Default'}`
also, we are comparing translated 'Default' to plain 'Default'

Reply:
Done